### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by ghost**
Adding code of conduct documentation as part of Hacktoberfest 2017.

### CODE REVIEW

**Comment by DanielCBrooks** [comment]
It looks like Continuous Integration is broken. A quick scan through the Travis CI logs reveals unsupported tokens in one of the node_modules dependencies, as well as an error related to an outdated version of Node (major version 3).

I do not believe this is related to my documentation add.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+SendGrid Community Code of Conduct
```
Comment (mbernier) [inline CODE_OF_CONDUCT.md]: Can you please add the markdown formatting that was present in the source file?

**Review by mbernier — DISMISSED** [review]

**Comment by DanielCBrooks** [comment]
Oh wow, huge miss there on my part. I've got it back in.

**Comment by thinkingserious** [comment]
Hello @DanielCBrooks,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by mbernier** [comment]
@DanielCBrooks 
If you are interested, we would love the opportunity to talk to you about Hacktoberfest and our API libraries.

Our agenda would be: Explore what you liked and is there anything we can do to improve? 

You can grab a time [on my calendar](https://calendly.com/matt-bernier/hacktoberfest-follow-up) that works for you and we can have a chat on Google Hangout or Skype. If you prefer, you can email me using my GitHub username at my company’s domain.

Thank you so much,

Matt Bernier - @mbernier - SendGrid Developer Experience Product Manager
Elmer Thomas - @thinkingserious - SendGrid Developer Experience Engineer
@SendGrid

**Pull request merged**